src/routes/tags/fetch-tags.e2e.test.js

```diff
@@ -2,48 +2,60 @@ const { expect } = require('chai')
 const { fixtures, resetDBTable } = require('../../../test/utils')
 
 describe('GET /tags', () => {   
+
 	let user
 	let accessToken
+
 	beforeEach(async ()=> {
+
 		user = await fixtures.insertUser()
 		const body = { id: user.id, email: user.email }
 		accessToken = fixtures.generateAccessToken(
 			{user : body}
 		)
 	})
+
 	describe('Failure', ()=> {
```

Comment:
Add a logger here. Similar to what is here:
https://github.com/tbf-challenge/teamwork-be/pull/300/files#diff-9d469a35ac5ab7213420d0e84ab4e3cf6d1105a603dd131a54bd85e0b0b89b53R85
 To investigate flaky test.

Reply:
sure

Reply:
@jocrah , i have implemented suggested changes now, kindly review